app/controllers/about.js

```diff
@@ -0,0 +1,14 @@
+import Controller from '@ember/controller';
+import { action } from '@ember-decorators/object';
+
+export default class AboutController extends Controller {
+  // for recent projects that don't have a carto row manually fly the user to the map
+  @action
+  flyTo(zoom, lat, lng) {
+    // global map object, set in applicationController
+    window.map.flyTo({
+      zoom,
+      center: { lng, lat },
+    });
```

Comment:
Let's find a different approach. Is `window.map` used anywhere else in the application? Often it was set on `window` (map) for debugging purposes.

We'll want to use services or something to access things like the map.

Reply:
@allthesignals I didn't know `window.map` was being set for debugging purposes, just saw that the `application.js` controller was setting [`window.map = map` in the `handleMapLoad` action](https://github.com/NYCPlanning/labs-waterfront-access/blob/001c1f69fff51921fe17edd08c17c04af854863d/app/controllers/application.js#L99.  Was making an assumption that this application wouldn't have a second map, so thought it'd be OK to just borrow from the global `window.map` defined object.

Revising PR now to use an Ember service instead.